### PR TITLE
Improve test coverage to 100%

### PR DESCRIPTION
**Pull request by elliotwutingfeng**
## Changes

- Added missing test cases. Test coverage is now **100%**.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.html
```
Comment (rivo) [inline .gitignore:1]: Please don't add `.gitignore` to this repo. It clashes with my (and potentially others') local `.gitignore` file. And if someone decides to generate coverage reports, it should be their own duty to clean them up before checking anything in. (I wouldn't accept pull requests with these files anyway.)

File: README.md [hunk 2]
```diff
@@ -41,6 +41,12 @@ Most terminals or text displays / text editors using a monospace font (for examp
 go get github.com/rivo/uniseg
 ```
 
+## Testing
+
+```bash
```
Comment (rivo) [inline README.md:46]: I don't think it is necessary to add this here. It doesn't provide much information about the repo. I would think that most people don't care about coverage. And if they do, they can look up in the Go docs how to do it.

So I think you should remove this change.

**Review by rivo — CHANGES_REQUESTED** [review]
First of all, thank you for your contribution. I have two comments, requests for change.

I don't really insist on 100% test coverage in this project so with future changes, this percentage might decrease again. But I'm happy for you to check from time to time whether we're still at 100%. (Although I don't expect too many changes/additions to this package anymore from this point on.)

File: .gitignore (hunk 1 above)
Comment (elliotwutingfeng) [reply .gitignore]: Done!

File: README.md (hunk 2 above)
Comment (elliotwutingfeng) [reply README.md]: Done!

**Pull request merged**